api/v1/series/SerieHandler.inc.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * @file api/v1/user/SerieHandler.inc.php
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2003-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SerieHandler
+ * @ingroup api_v1_user
+ *
+ * @brief Handle API requests for user operations.
+ *
+ */
+
+import('lib.pkp.classes.handler.APIHandler');
+import('classes.core.ServicesContainer');
+
+class SerieHandler extends APIHandler {
```

Comment:
I think this should be SeriesHandler throughout (e.g. `seriesId` rather than `serieId`).